client.go

```diff
@@ -30,8 +30,12 @@ func (h *httpHeader) SetHeader(header http.Header) {
 	*h = httpHeader(header)
 }
 
-func (h httpHeader) Header() http.Header {
-	return http.Header(h)
+func (h *httpHeader) Header() http.Header {
```

Comment:
Quick question: why do we switch to the pointer type here?

Reply:
> Quick question: why do we switch to the pointer type here?

Because my IDE Goland warns: "Struct httpHeader has methods on both value and pointer receivers. Such usage is not recommended by the Go Documentation."